### ISSUE TITLE
There is no `flux reconcile source chart`

### ISSUE THREAD

**Issue by kingdonb** [fluxcd/flux2#3656]
When a configuration like this one:

https://github.com/kingdon-ci/fleet-infra/blob/663937f2414a670331c363766cd8df501400db7c/clusters/test/limnocentral/wasm-test/stage/taking-bart-helmrelease.yaml#L12-L15

is linked to a `HelmRepository` like this one:

https://github.com/kingdon-ci/fleet-infra/blob/663937f2414a670331c363766cd8df501400db7c/clusters/test/limnocentral/wasm-test/stage/taking-bart-helmrepo.yaml#L13-L21

and a new chart version is published, it will not be picked up until the `HelmChart` is reconciled. On what interval that gets reconciled, I think you can configure in `spec.chart.spec.interval` but there is no `flux reconcile source chart` to reconcile this on-demand.

I'm not sure that's what is needed exactly (maybe I should set up webhooks, and point the right event at my `HelmChart` with an `Alert` – who wants to reconcile things by hand, or even tweak some random interval for that matter?)

But it seems like a logistical gap that there is no way in the CLI to reconcile the chart by hand, if you may sometimes need to do it.

**Comment by stefanprodan** [comment]
Yes this is needed for OCI as reconciling the Helm repo does nothing.
